examples/3d-tiles/app.js

```diff
@@ -267,8 +284,10 @@ export default class App extends PureComponent {
       coordinateOrigin,
       depthLimit,
       color,
-      onTileLoaded: this._onTileLoaded.bind(this),
-      onTilesetLoaded: this._onTilesetLoaded.bind(this)
+      onTilesetLoaded: this._onTilesetLoaded,
```

Comment:
Nit: our naming convention is `on<verb>`, e.g. `onViewStateChange`, `onClick`, `onDataLoad`

Reply:
> Nit: our naming convention is on<verb>, e.g. onViewStateChange, onClick, onDataLoad

https://github.com/uber-web/loaders.gl/pull/336